homeassistant/components/wilight/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "config": {
+    "flow_title": "WiLight: {name}",
+    "step": {
+      "confirm": {
+        "title": "WiLight",
+        "description": "Do you want to set up WiLight {name}?\n\n It supports: {components}"
+      }
+    },
+    "abort": {
+      "already_in_progress": "WiLight configuration is already in progress.",
+      "already_configured": "This WiLight is already configured.",
```

Comment:
Please use as many references as possible. Here they are:
https://github.com/home-assistant/core/blob/24a16ff8febff2040c00b83b42b04e02233854fa/homeassistant/strings.json#L19-L58

Here's how to reference:
https://github.com/home-assistant/core/blob/111c2006c8cf37a0d26995c0048c26780f731017/homeassistant/components/adguard/strings.json#L7

Reply:
I reviewed the strings along with config_flow.py and removed some unnecessary ones